generator/AWSPSGeneratorLib/HelpMaterials/Examples/IAM/Get-IAMServiceLastAccessedDetailWithEntity.xml

```diff
@@ -0,0 +1,29 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--Examples for the AWS PowerShell CmdLets-->
+<examples>
+  <example>
+    <code>$results = Get-IAMServiceLastAccessedDetailWithEntity -JobId f0b7a819-eab0-929b-dc26-ca598911cb9f -ServiceNamespace ec2
+$results
+Output:
+EntityDetailsList : {Amazon.IdentityManagement.Model.EntityDetails}
+Error             : 
+IsTruncated       : False
+JobCompletionDate : 12/29/19 11:19:31 AM
+JobCreationDate   : 12/29/19 11:19:31 AM
+JobStatus         : COMPLETED
+Marker            : 
+$results.EntityDetailsList
+Output:
+EntityInfo                                 LastAuthenticated
+----------                                 -----------------
+Amazon.IdentityManagement.Model.EntityInfo 11/16/19 3:47:00 PM
+$results.EntityInfo
```

Comment:
For multi-line samples the generator won't add the `PS C:\> ` prefix automatically (it doesn't know which lines are commands and which are output). So, especially in case you have multiple commands, it would be better to prefix them with `PS C:\> ` to help distinguishing them from their output.